src/ext/draft/tuning.rs

```diff
@@ -0,0 +1,55 @@
+use crate::{events::*, host::*, id::*, plugin::*, string_sizes::*};
+
+use std::os::raw::c_char;
+
+pub const CLAP_EXT_TUNING: *const c_char = b"clap.tuning.draft/2\0".as_ptr() as *const c_char;
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct clap_event_tuning {
+    pub header: clap_event_header,
+    pub port_index: i16,
+    pub channel: i16,
+    pub tunning_id: clap_id,
+}
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct clap_tuning_info {
+    pub tuning_id: clap_id,
+    pub name: [c_char; CLAP_NAME_SIZE],
+    pub is_dynamic: bool,
+}
+
+unsafe impl Send for clap_tuning_info {}
+unsafe impl Sync for clap_tuning_info {}
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct clap_plugin_tuning {
```

Comment:
The original struct is called `clap_client_tuning`, although that's probably a mistake: https://github.com/free-audio/clap/blob/cf2bdb1380d97dcf6d69ba603c444b1981efcd3d/include/clap/ext/draft/tuning.h#L30

Reply:
Huh, you're right. I'll 'fix' it and create an issue on the CLAP repo.

Reply:
Seems like I added this in the CLAP 0.26 PR (since both kind of came at the same time) but I forgot to add that change to this PR.